File: PS 1/mario-more/mario.c
#include <cs50.h>
#include <stdio.h>

int height;
int r;
int c;
int s;
int main(void)
{


    do

    {

        //Ask the user for the height
        height = get_int("What's the height? \n");

    }
    //loop for height between 1 to 8
    while (height > 8 || height < 1);
    //loop for rows
    for (r = 0 ; r < height ; r++)
    {
        //loop for spaces
        for (s = 0 ; s < height - r - 1; s++)
        {
            printf(" ");
        }
        //loop for colums
        for (c = 0 ; c <= r ; c++)
        {
            printf("#");
        }


        printf("  ");


        //loop for colums
        for (c = 0 ; c <= r ; c++)
        {
            printf("#");
        }


        printf("\n");


    }


}

File: PS 6/sentimental-mario-more/mario.py
# TODO
from cs50 import get_int


def main():

    while True:
        height = get_int("Height: ")
        if height >= 1 and height <= 8:
            break

    for r in range(1, height + 1):
        print(" " * (height - r) + "#" * r + "  " + "#" * r)

    # for r in range(1, height + 1):
    #     for s in range(0,height - r):
    #         print(" ", end="")
    #     for c in range(0, r):
    #         print("#", end="")
    #     print("  ", end="")
    #     for c in range(0, r):
    #         print("#", end="")
    #     print("")


if __name__ == "__main__":
    main()